client/src/pages/UsersPage.tsx

```diff
@@ -0,0 +1,119 @@
+import API from "@aws-amplify/api";
+import { TextField } from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+import { Table } from "react-bootstrap";
+import Loader from "../components/global/Loader";
+
+const usersHeaders = [
+  "Name",
+  "Student Number",
+  "Hours Completed",
+  "Hours Required",
+  "Checked In",
+];
+
+export const UsersPage = () => {
+  const [loading, setLoading] = useState(true);
+  const [usersCheckedIn, setUsersCheckedIn] = useState(0);
+  const [users, setUsers] = useState(null);
+  const [query, setQuery] = useState("");
+
+  useEffect(() => {
+      // TODO: Sort users by name alphabetically
+    loadUsers();
+  }, [query]);
```

Comment:
Having [query] in the dependency array is telling react that every time the value of query changes, let's call `loadUsers` again and update the UI. The query isn't being used in the API call, but will be after #51 is done.